### DISCUSSION TITLE
Backend API data not fetch in frontend

### DISCUSSION THREAD

**Discussion by PazhaniMani** [blockscout/blockscout#8786] [Support]
I follow this step [for](https://docs.blockscout.com/for-developers/information-and-settings/docker-integration-local-use-only) local macOS 
and run frontend by this command 
->> docker run -p 3000:3000 --env-file '/Users/pcodeindia/Desktop/blockscout/docker-compose/envs/common-frontend.env' ghcr.io/blockscout/frontend:latest

frontend working...  [this url](http://localhost:3000) but backend data not showing frontend why

![Screenshot 2023-11-06 at 2 10 00 PM](https://github.com/blockscout/blockscout/assets/69962094/81bb46d4-1a24-4324-a218-257f5ca6e404)

I attach the screen shot above, please help me. how to fix it. and detail explanation step by step guide

Thankyou......

**Comment by ekawibisono** [comment]
Your backend url has been blocked CORS, i think solution install plugin enable CORS in your browser extension.
